### PR TITLE
Questions initial implementation

### PR DESCRIPTION
**Pull request by PodviaznikovPavlo**
@podviaznikov 

I have pushed new version. Here is a greeting window. When you enter your name, the main window will appear. All online users logging to condole now(in browser). 
Tomorrow i'll do the greeting from bot to new user and online users.
But i have a question about disconnected users. As you can see, i'm pushing new users to array, which i get on front and displaying, but how can i get the user, who went offline and remove him from array?

Well, thats all for today.

### CODE REVIEW

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 

I have finished task 'online users' and 'greetings from bot'. Check it and tell me.

**Comment by podviaznikov** [comment]
'but how can i get the user, who went offline and remove him from array?'.

Try to research it. There are solutions. Try to find yourself.

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 

I found the solution. This task is ready!!!

File: sh.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,24 @@
+MSYS-1.0.12 Build:2012-07-05 14:56
```
Comment (podviaznikov) [inline sh.exe.stackdump]: what is this file? remove it from git.

**Comment by podviaznikov** [comment]
Nice that you find solution!

File: questions.js [hunk 2]
```diff
@@ -1,74 +1,46 @@
-'use strict';
-var questions = [],
-    fs = require('fs'),
-	JSON5 = require('json5'),
-    correctNameEnding,
-    jsonData;
 
-var originCountries = ['Україна', 'Англія', 'Франція', 'Іспанія', 'Німеччина', 'Чехія', 'Молдова', 'Уельс', 'Китай', 'Азербайджан', 'Чилі'];
```
Comment (podviaznikov) [inline questions.js:8]: why did you remove this code? We still need to do this.

You can create your test questions in the different file.
